Automattic/wp-calypso
# Tailored Flows: Link in bio processing step

## Proposed Changes

Add processing step for link in bio flow.

<img width="1303" alt="image" src="https://user-images.githubusercontent.com/2653810/184618917-5409a3b2-0146-4578-a431-9edf2edf27f1.png">

<img width="410" alt="image" src="https://user-images.githubusercontent.com/2653810/184618952-7cb9878e-b10d-486c-b4d7-96c08bbc943c.png">

## Testing instructions

- Pull this branch and run `yarn start` from the root folder
- Navigate to `http://calypso.localhost:3000/setup/intro?flow=link-in-bio&siteSlug=YOURSITE.wordpress.com`
- Advance on the steps by filling the fields
- After clicking on continue, you should see the processing step with the designs above.


Related to #66561 
Fixes #66561
